config/nginx/django.conf

```diff
@@ -6,6 +6,39 @@ upstream web {
 server {
```

Comment:
@akmiller01 
shouldn't this redirect to https then?

Reply:
It upsets Django a bit if you make Nginx do the redirect instead of WSGI. So the SECURE_SSL_REDIRECT is set in settings.py

Reply:
cool